### PR TITLE
HY-4964 Turn on strong mode and lint rules

### PR DESCRIPTION
**Pull request by robbecker-wf**
Turn on strong mode and add required lint rules.

# Testing
I pulled this into w_transport and all tests pass
image

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Raven
Number of Findings: 0

**Comment by robbecker-wf** [comment]
@charliekump-wf @evanweible-wf @davidmarne-wf someone review and merge

File: example/echo.dart [hunk 1]
```diff
@@ -2,7 +2,7 @@ library echo;
 
 import "dart:html";
 import 'package:logging/logging.dart';
-import "package:sockjs_client/sockjs.dart" as SockJS;
+import "package:sockjs_client/sockjs_client.dart" as SockJS;
```
Comment (evanweible-wf) [inline example/echo.dart]: Was this just straight up broken?

Comment (robbecker-wf) [inline example/echo.dart]: it was totally the wrong path .. i think the example must not have been able to work.

**Comment by trentgrover-wf** [comment]
@jayudey-wf ready for merge

**Comment by jayudey-wf** [comment]
QA +1

- [ ] Testing instruction
- [ ] Dev +1's
- [ ] Dev/QA +10 with detail of what was tested
- [ ] Security review (if required)
- [ ] Unit tests created/updated - N/A dep bump
- [ ] All unit tests pass
- [ ] Rosie has run and reports properly the release the ticket will be included in

Merging into master.

**Comment by robbecker-wf** [comment]
@evanweible-wf Do we need to cut a release?

**Pull request merged**